internal/nutanix/storage_containers.go

```diff
@@ -147,7 +149,24 @@ func (e *StorageContainerExporter) Collect(ch chan<- prometheus.Metric) {
 		key := KEY_STORAGE_CONTAINER_PROPERTIES
 		var property_values []string
 		for _, property := range e.properties {
-			val := fmt.Sprintf("%v", ent[property])
+			var val string = ""
+			// format properties
+			switch property {
+			case "max_capacity_mb":
+				propname := "max_capacity_mb"
```

Comment:
This seems incorrect... Should this be just "max_capacity'? Please test the changes throughly

Reply:
push again

Reply:
Would you share the output... Output should have max_capacity_mb right?